GitCommands/Git/GitItemStatus.cs

```diff
@@ -237,6 +237,30 @@ internal void SetSubmoduleStatus(JoinableTask<GitSubmoduleStatus?> status)
             _submoduleStatus = status;
         }
 
+        /// <summary>
+        /// Return an object with the status as if the item was created
+        /// with first and second commit reverse selected.
+        /// </summary>
+        /// <returns>An inverted copy.</returns>
+        public GitItemStatus ReverseSelection()
```

Comment:
I'm not finding the signature meaningful - it's quite unexpected for a method called "reverse selection" to return "item status". Maybe call something like `InvertStatus()`?

Reply:
Works, changing